res/values/strings.xml

```diff
@@ -82,6 +82,10 @@
     <string name="file_list_file">file</string>
     <string name="file_list_files">files</string>
     <string name="filedetails_select_file">Tap on a file to display additional information.</string>
+    <string name="upload_owncloud"> Upload to OwnCloud </string>
+    <string name="upload_bottom_sheet_displayed"> Upload Bottom Sheet Displayed </string>
```

Comment:
This resource is not being used, please delete it

Reply:
Actually I am using upload_owncloud

Reply:
I meant upload_bottom_sheet_displayed, which you've already deleted. 

There're still some space at the beginning and end of *Upload to ownCloud* in upload_owncloud resource. Do you really need it? If so, please use margin or padding in the xml layout.